lerna.json

```diff
@@ -1,4 +1,4 @@
 {
-	"packages": [ "packages/*", "client/gutenberg/extensions/presets/jetpack" ],
+	"packages": [ "packages/*" ],
```

Comment:
by hand for now. Since they're not really using semver versions of things from inside calypso, it's very hard / impossible for lerna to handle the versions automatically.